src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1026,8 +1028,7 @@ void logic_PropertyChanged(object sender, PropertyChangedEventArgs e)
                 case "IsVisible":
                     RaisePropertyChanged("IsVisible");
                     RaisePropertyChanged(nameof(NodeWarningBarVisible));
-                    if (ErrorBubble != null) return;
```

Comment:
could this cause an issue where the preview off color will override the error color?

Reply:
I have the same question, it was there on that purpose

Reply:
The color was being set previously as a function of the ErrorBubble.  This doesn't actually matter.  The state of the error bubble only controls the visibility of the WarningBar.  The color of the warning bar is a function of the state of the node.  Error -> Red, Warning or Persistent Warning -> Yellow, Default -> Grey.  Even if this event trigger the update color, the logic pin the `GetWarningColor` will never let it be set to Grey.